core/src/main/java/org/zenoss/zep/index/impl/lucene/LuceneQueryBuilder.java

```diff
@@ -386,6 +386,9 @@ private void buildPathFilters(List<Filter> filters, String fieldName, Set<String
                 // Prefix
                 else if (value.endsWith("*")) {
                     value = value.substring(0, value.length() - 1);
+                    if (value.endsWith("*")) {
+                        value = value.substring(0, value.length() - 1);
+                    }
```

Comment:
Are two asterisks the most that could ever appear at the end of an string?

Perhaps an algorithm that collapsed any number of consecutive asterisks into one asterisk would provide better long-term benefit.